packages/client/src/merchantsLocations/__fixtures__/getMerchantsLocations.fixtures.ts

```diff
@@ -1,34 +1,20 @@
-import join from 'proper-url-join';
-import moxios from 'moxios';
-import type { GetMerchantsLocationsQuery, MerchantLocation } from '../types';
+import { rest, RestHandler } from 'msw';
+import type { MerchantLocation } from '../types';
+
+const path = '/api/commerce/v1/merchantsLocations';
 
-/**
- * Response payloads.
- */
 export default {
-  success: (params: {
-    query: GetMerchantsLocationsQuery;
-    response: MerchantLocation[];
-  }): void => {
-    moxios.stubRequest(
-      join('/api/commerce/v1/merchantsLocations', {
-        query: params.query,
-      }),
-      {
-        response: params.response,
-        status: 200,
-      },
-    );
-  },
-  failure: (params: { query: GetMerchantsLocationsQuery }): void => {
-    moxios.stubRequest(
-      join('/api/commerce/v1/merchantsLocations', {
-        query: params.query,
-      }),
-      {
-        response: 'stub error',
-        status: 404,
-      },
-    );
-  },
+  success: (response: MerchantLocation[]): RestHandler =>
+    rest.get(path, async (req, res, ctx) =>
+      res(ctx.status(200), ctx.json(response)),
+    ),
+  failure: (): RestHandler =>
+    rest.get(path, async (req, res, ctx) =>
+      res(
+        ctx.status(404),
+        ctx.json({
+          message: 'stub error',
+        }),
+      ),
```

Comment:
Is the formatting on this chunk okay? Shouldn't all this be in one line?
```suggestion
      res(ctx.status(404), ctx.json({ message: 'stub error' })),
```

Reply:
The formatting was okay but I removed the last comma so it can be inline :)